### PR TITLE
Adding drone CLI to KDK

### PR DESCRIPTION
**Pull request by prabdeb**
Adding Drone CLI to KDK from http://docs.drone.io/cli-installation/

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -389,3 +389,11 @@
   yum:
     name: "{{ item.name }}"
   with_items: "{{ yum_packages }}"
+
+- name: Drone client
+  tags:
+    - drone
+  unarchive:
+    src: http://downloads.drone.io/release/linux/amd64/drone.tar.gz
```
Comment (darkyat) [inline tasks/main.yml]: This shouldn't be hardcoded but using varilables. Both URL and destination should be set in defaults/main.yml. As an example see L239 `mc | Install latest binary release.`

File: tasks/main.yml [hunk 2]
```diff
@@ -389,3 +389,11 @@
   yum:
     name: "{{ item.name }}"
   with_items: "{{ yum_packages }}"
+
```
Comment (darkyat) [inline tasks/main.yml]: Please also add a sha256sum check like in L224. You can find the sha256 sum of the latest drone build here: http://downloads.drone.io/release/linux/amd64/drone.sha256

**Comment by prabdeb** [comment]
@darkyat Thanks! updated it now.

**Comment by prabdeb** [comment]
Thanks @dcwangmit01 I have changed the order now!

**Comment by dcwangmit01** [comment]
@prabdeb Merged your PR. It'd be great if you could do the same for the dockerized KDK.  This KDK based on vagrant is about to be deprecated.

**Pull request merged**